example/.eslintrc

```diff
@@ -2,7 +2,7 @@
     "extends": "eslint-config-nfl",
     "env": {
         "browser": true,
-        "mocha": true
+        "jasmine": true
```

Comment:
This change is causing some lint warnings. Can you update `context` -> `describe`?

```
> react-wildcat-example@7.1.1 lint wildcat/example
> eslint src/* --cache --cache-location .cache/.eslintcache


wildcat/example/src/test/e2e/flows/indexPageFlow.js
  32:5  warning  'context' is not defined  no-undef
  33:9  warning  'context' is not defined  no-undef
  48:9  warning  'context' is not defined  no-undef

wildcat/example/src/test/e2e/flows/prefetchPageFlow.js
  27:5  warning  'context' is not defined  no-undef
  28:9  warning  'context' is not defined  no-undef
  43:9  warning  'context' is not defined  no-undef

✖ 6 problems (0 errors, 6 warnings)
```